Index: src/main/java/com/laola/apa/utils/DateUtils.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/laola/apa/utils/DateUtils.java	(revision ae034b453523222709e6714ec31d4a0aa3f582a3)
+++ src/main/java/com/laola/apa/utils/DateUtils.java	(date 1619518071721)
@@ -186,7 +186,7 @@
         return endSum /  endTimes-  beganSum / beginTimes;
     }
 
-    /**
+    /** 
      * 取得最后一点
      *
      * @param mapList
